examples/3d/update_gltf_scene.rs

```diff
@@ -37,56 +25,49 @@ fn setup(
         ..default()
     });
 
-    // Spawn the scene as a child of another entity. This first scene will be translated backward
-    // with its parent
-    commands
-        .spawn_bundle(TransformBundle::from(Transform::from_xyz(0.0, 0.0, -1.0)))
-        .with_children(|parent| {
-            parent.spawn_scene(asset_server.load("models/FlightHelmet/FlightHelmet.gltf#Scene0"));
-        });
+    // Spawn the scene as a child of this entity at the given transform
+    commands.spawn_bundle(SceneBundle {
+        transform: Transform::from_xyz(0.0, 0.0, -1.0),
+        scene: asset_server.load("models/FlightHelmet/FlightHelmet.gltf#Scene0"),
+        ..default()
+    });
 
     // Spawn a second scene, and keep its `instance_id`
-    let instance_id =
-        scene_spawner.spawn(asset_server.load("models/FlightHelmet/FlightHelmet.gltf#Scene0"));
-    scene_instance.0 = Some(instance_id);
+    commands
+        .spawn_bundle(SceneBundle {
+            scene: asset_server.load("models/FlightHelmet/FlightHelmet.gltf#Scene0"),
+            ..default()
+        })
+        .insert(MovedScene);
 }
```

Comment:
Looks like this comment is no longer relevant?

Reply:
updated 👍